hawkbit-runtime/docker/docker-compose-stack.yml

```diff
@@ -79,8 +79,8 @@ services:
         condition: on-failure
     environment:
       MYSQL_DATABASE: "hawkbit"
-      # MYSQL_USER: "root" is created by default in the container for mysql 5.7+
-      MYSQL_ALLOW_EMPTY_PASSWORD: "true"
+      MYSQL_USER: "root"
```

Comment:
Hey @rojarsmith,
thanks for your contribution. It looks like `MYSQL_ALLOW_EMPTY_PASSWORD: "true"` was moved into the wrong line. Furthermore `MYSQL_USER` can not be set here because otherwise we will get again the same error that was already fixed by https://github.com/eclipse/hawkbit/pull/1144

Reply:
Hi, I am sorry don't know how to pull request for one commit:

https://github.com/eclipse/hawkbit/commit/cf086acdd21f9602fa18b10ef5156326b303d011

Reply:
Please make sure to update your fork to the latest changes of hawkbit. Afterwards you can change only the code you want to e.g. https://github.com/eclipse/hawkbit/commit/cf086acdd21f9602fa18b10ef5156326b303d011. Please update this pull request accordingly.